### DISCUSSION TITLE
Global fetchReply option for Client

### DISCUSSION THREAD

**Discussion by Drylozu** [discordjs/discord.js#6474] [Ideas]
The idea is that `fetchReply` from `MessageComponentInteraction`/`BaseCommandInteraction` `#reply` (and others) can be global and specified on the `Client`, as well as `allowedMentions` and not having to set it for every `interaction.reply`.

The idea written in code would be something like:
```js
// Main file
const { Client } = require('discord.js');
new Client({
  intents: [...],
  fetchReplies: true
});


// interactionCreate event
interaction.reply({
  content: 'upa'
}); // This will return a Message or APIMessage because fetchReplies is `true` in the client options
```

**Comment by monbrey** [comment]
This could work now that it's possible to fetch ephemerals too, though I'd still prefer if the API itself could return the data and drop the option completely.
